arch/x86/kvm/pkvm/pgtable.h

```diff
@@ -165,4 +166,33 @@ static inline enum pkvm_page_state pkvm_pte_pgstate(struct pkvm_pgtable *pgt, vo
 	return pgt->pgt_ops->pte_pgstate(ptep);
 }
 
+/*
+ * Use bit31 ~ bi12 as owner ID bits to avoid conflicting w/ low 12 property
+ * bits for all kinds of page table.
```

Comment:
Just a note: I can imagine page tables where this doesn't hold. E.g. on ARM with 16K or 64K pages. :)

Reply:
Not familiar with ARM with 16K or 64K pages, do you think for this kind of case, we cannot use bit12 ~ bit31 to store owner ID, as some bits in this range will be property bits for such page table?

Reply:
The owner ID is stored in non-present PTE. And avoid using the low 12 bits is to make sure the stored owner ID won't be wrongly recognized as a present PTE. So for 16K or 64K pages table, as long as they don't use the bits above bit12 as the present bit, then we should be fine.